Doc/library/ctypes.rst

```diff
@@ -274,6 +274,10 @@ complex types are available:
 +==================================+=================================+=================+
 | :class:`c_double_complex`        | :c:expr:`double complex`        | complex         |
 +----------------------------------+---------------------------------+-----------------+
+| :class:`c_float_complex`         | :c:expr:`float complex`         | complex         |
++----------------------------------+---------------------------------+-----------------+
+| :class:`c_longdouble_complex`    | :c:expr:`long double complex`   | complex         |
++----------------------------------+---------------------------------+-----------------+
```

Comment:
This may be nitpicking, but I would have `c_float_complex` *before* `c_double_complex` (in the above table, *double* is after *float*).

```suggestion
+==================================+=================================+=================+
| :class:`c_float_complex`         | :c:expr:`float complex`         | complex         |
+----------------------------------+---------------------------------+-----------------+
| :class:`c_double_complex`        | :c:expr:`double complex`        | complex         |
+----------------------------------+---------------------------------+-----------------+
| :class:`c_longdouble_complex`    | :c:expr:`long double complex`   | complex         |
+----------------------------------+---------------------------------+-----------------+
```